### PR TITLE
Move Rafter docs

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Since Rafter will not be part of Kyma 2.0, we are moving the docs from the `kyma` repo to the native `rafter` repository.
The docs will be removed from the `kyma` repository once the new docs structure is introduced.
Changes proposed in this pull request:

- Moved Rafter docs from the `kyma` to the `rafter` repo.
- Changed numbering
- Corrected links

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/10927

### CODE REVIEW

File: docs/03-rafter-in-kyma.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (kwiatekus) [inline docs/03-rafter-in-kyma.md:1]: IMO this content is not needed anymore.

File: docs/10-minio-gateway.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: MinIO and MinIO Gateway
+type: Details
+---
+
+The whole concept of Rafter relies on MinIO as the storage solution. It supports Kyma's manifesto and the "batteries included" rule by providing you with this on-premise solution by default.
```
Comment (kwiatekus) [inline docs/10-minio-gateway.md:6]: describe usage scenarios w/o referencing to kyma.

File: docs/10-minio-gateway.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+![MinIO Gateway](./assets/minio-gateway.svg)
+
+>**TIP:** Using Gateway mode may generate additional costs for storing buckets, assets, or traffic in general. To avoid them, verify the payment policy with the given cloud provider before you switch to Gateway mode.
+
+See [this tutorial](./22-set-minio-to-gateway-mode.md) to learn how to set MinIO to Google Cloud Storage Gateway mode.
+
+## Access MinIO credentials
+
+For security reasons, MinIO credentials are generated during Kyma installation and stored inside the Kubernetes Secret object. You can obtain both the access key and the secret key in the development (MinIO) and production (MinIO Gateway) mode using the same commands.
```
Comment (kwiatekus) [inline docs/10-minio-gateway.md:39]: Are those credentials generated simply at installation time of the helm chart ? ( lets be runtime agnostic )

File: docs/14-front-matter-service.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+ - foo
+ - bar
+---
+
+## Lorem ipsum
+Dolores sit amet
+
+```
+
+## Use the service outside the Kyma cluster
```
Comment (kwiatekus) [inline docs/14-front-matter-service.md:30]: Exposing fron-matter service via port-forward

File: docs/13-asyncapi-service.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: AsyncAPI Service
+type: Details
+---
+
+AsyncAPI Service is an HTTP server enabled by default in Kyma to process AsyncAPI specifications. It only accepts `multipart/form-data` forms and contains two endpoints:
```
Comment (kwiatekus) [inline docs/13-asyncapi-service.md:6]:
```suggestion
AsyncAPI Service is an HTTP server to process AsyncAPI specifications. It only accepts `multipart/form-data` forms and contains two endpoints:
```

File: docs/12-upload-service.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+To enable the service scaling and to maintain the bucket configuration data between the application restarts, the Upload Service stores its configuration in the `rafter-upload-service` ConfigMap.
+
+Once you upload the files, system buckets store them permanently. There is no policy to clean system buckets periodically.
+
+The diagram describes the Upload Service flow:
+
+![Upload Service](./assets/upload-service.svg)
+
+## Use the service outside the Kyma cluster
```
Comment (kwiatekus) [inline docs/12-upload-service.md:23]:
```suggestion
## Exposing upload service via port-forward
```

File: docs/10-minio-gateway.md (hunk 3 above)
Comment (kazydek) [reply docs/10-minio-gateway.md:39]: @tgorgol Could you advise how to make this section more runtime-agnostic?

File: docs/14-front-matter-service.md (hunk 4 above)
Comment (kazydek) [reply docs/14-front-matter-service.md:30]: `Expose the service using port-forward` (I changed that for the Upload Service as well)

File: docs/13-asyncapi-service.md (hunk 5 above)
Comment (kazydek) [reply docs/13-asyncapi-service.md:6]: or `AsyncAPI Service is an HTTP server that processes AsyncAPI specifications.`

File: docs/14-rafter-chart-configuration.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values.
+
+>**NOTE:** You can define all **envs** either by providing them as inline values or using the **valueFrom** object. See the [example](../charts/rafter-controller-manager#change-values-for-envs-parameters) for reference.
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **controller-manager.minio.persistence.enabled** | Parameter that enables MinIO persistence. Deactivate it only if you use [Gateway mode](#tutorials-set-minio-to-gateway-mode). | `true` |
```
Comment (klaudiagrz) [inline docs/14-rafter-chart-configuration.md:18]: Please fix the link

File: docs/17-assetgroup-cr.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+## Custom resource parameters
+
+This table lists all possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Required      |  Description |
+|----------|:-------------:|------|
+| **metadata.name** | Yes | Specifies the name of the CR. It also defines the **rafter.kyma-project.io/asset-group** label added to the Asset CR that the AssetGroup CR defines. Because of label name limitations, AssetGroup CR names can have a maximum length of 63 characters. |
+| **metadata.labels** | No | Specifies how to filter and group Asset CRs that the AssetGroup CR defines. See the [details](/components/console/#details-rafter-in-the-console) to learn more about these labels. |
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/17-assetgroup-cr.md:61]: I removed the link as it was no longer valid.

File: docs/18-clusterassetgroup-cr.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+## Custom resource parameters
+
+This table lists all possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Required      |  Description |
+|----------|:-------------:|------|
+| **metadata.name** | Yes | Specifies the name of the CR. It also defines the **rafter.kyma-project.io/asset-group** label added to the ClusterAsset CR that the ClusterAssetGroup CR defines. Because of label name limitations, ClusterAssetGroup CR names can have a maximum length of 63 characters. |
+| **metadata.labels** | No | Specifies how to filter and group ClusterAsset CRs that the ClusterAssetGroup CR defines. See the [details](/components/console/#details-rafter-in-the-console) to learn more about these labels. |
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/18-clusterassetgroup-cr.md:55]: I removed the link as it was no longer valid.

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**